src/index.js

```diff
@@ -281,35 +254,36 @@ const checkoutStartedCompletedEventMapping = [
           body.insertBefore(scriptTag, body.firstChild);
         }
       };
-      window.rudderAnalyticsMount = function () {
-        if (typeof globalThis === "undefined") {
-          Object.defineProperty(Object.prototype, "__globalThis_magic__", {
-            get: function get() {
-              return this;
-            },
-            configurable: true,
-          });
-          __globalThis_magic__.globalThis = __globalThis_magic__;
-          delete Object.prototype.__globalThis_magic__;
-        }
-        window.rudderAnalyticsAddScript(
-          ""
-            .concat(sdkBaseUrl, "/")
-            .concat(window.rudderAnalyticsBuildType, "/")
-            .concat(sdkName),
-          "data-rsa-write-key",
-          WRITE_KEY,
-        );
+      window.rudderAnalyticsMount = function() {
+        (function() {
+          if (typeof globalThis === "undefined") {
+            var getGlobal = function getGlobal() {
+              if (typeof self !== "undefined") {
+                return self;
+              }
+              if (typeof window !== "undefined") {
+                return window;
+              }
+              return null;
+            };
+            var global = getGlobal();
+            if (global) {
+              Object.defineProperty(global, "globalThis", {
+                value: global,
+                configurable: true
+              });
+            }
+          }
+        })();
+          window.rudderAnalyticsAddScript("".concat(sdkBaseUrl, "/").concat(window.rudderAnalyticsBuildType, "/").concat(sdkName), "data-rsa-write-key", WRITE_KEY);
       };
       if (typeof Promise === "undefined" || typeof globalThis === "undefined") {
-        window.rudderAnalyticsAddScript(
-          "https://polyfill-fastly.io/v3/polyfill.min.js?version=3.111.0&features=Symbol%2CPromise&callback=rudderAnalyticsMount",
-        );
+        window.rudderAnalyticsAddScript("https://polyfill-fastly.io/v3/polyfill.min.js?version=3.111.0&features=Symbol%2CPromise&callback=rudderAnalyticsMount");
       } else {
         window.rudderAnalyticsMount();
       }
       var loadOptions = {};
-      rudderanalytics.load(WRITE_KEY, DATAPLANE_URL, loadOptions);
+        rudderanalytics.load(WRITE_KEY, DATAPLANE_URL, loadOptions);
```

Comment:
I'll fix this in the snippet for all later. Can we please fix it here?
```suggestion
      rudderanalytics.load(WRITE_KEY, DATAPLANE_URL, loadOptions);
```

Reply:
fixed